test/integration/IntegrationTests.spec.ts

```diff
@@ -285,10 +267,25 @@ describe("Upload DAR integration test", () => {
         }
     };
 
-    it('should successfully submit and wait for a command', (done) => {
+    it('should be able to upload Dar file', (done) =>{
+        const darToUploadInString = darToUpload.toString('base64');
         withLedgerClient((client) => {
-            client.commandClient.submitAndWait(commands, (error) => {
+            client.packageManagementClient.uploadDarFile({
+                darFile: darToUploadInString
+            });
+
+            client.packageManagementClient.listKnownPackages((error, response) => {
                 expect(error).to.be.null;
+
+                if (response !== undefined) {
+                    response.packageDetailsList.forEach( (item) => {
+                        expect(item.packageId).to.be.not.null;
+                    })
+                }
+
+                client.commandClient.submitAndWait(commands, (error) => {
+                    expect(error).to.be.null;
```

Comment:
The call to `done` should be here, otherwise the test will pass before this happens.

Reply:
See next commit